install/09_alertrules.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: clusterautoscaler-rules
+  namespace: openshift-machine-api
+spec:
+  groups:
+    - name: general.rules
+      rules:
+        - alert: ClusterAutoscalerUnschedulablePods
+          expr: |
+            cluster_autoscaler_unschedulable_pods_count{service="cluster-autoscaler"} > 0
+          for: 20m
+          labels:
+            severity: critical
+          annotations:
+            message: "Cluster Autoscaler has {{ $value }} unschedulable pods"
+
+        - alert: ClusterAutoscalerNodesNotReady
+          expr: |
+            cluster_autoscaler_nodes_count{state!="ready"} > 0
+          for: 20m
+          labels:
+            severity: critical
+          annotations:
+            message: "Cluster Autoscaler has {{ $value }} unready nodes"
+
+        - alert: ClusterAutoscalerNotSafeToScale
+          expr: |
+            cluster_autoscaler_cluster_safe_to_autoscale{service="cluster-autoscaler"} != 1
+          for: 15m
+          labels:
+            severity: critical
```

Comment:
All of these alerts are cause based, not symptom based. Paging should happen on symptom based factors such as "Users can't schedule pods", these all seem like "warning" type alerts at most, maybe even "info" level.